tests/integration/components/did-insert-parent-hook-test.js

```diff
@@ -68,18 +77,19 @@ test('top-level parent and two children-parents', function(assert) {
   assert.ok(parentSpy.calledBefore(childSpy), 'parent was called before child');
 });
 
-test('top-level parent and two children-parents after if', function(assert) {
-  let parentSpy = this.parentSpy = this.spy();
-  let childSpy = this.childSpy = this.spy();
-  let childParentSpy = this.childParentSpy = this.spy();
+test('top-level parent with children-parents after if', function(assert) {
```

Comment:
Sorry for the delayed response, @reidab. Yes please, that would help a lot for me to understand what is going on here!